src/main/groovy/com/bugsnag/android/gradle/BugsnagVariantOutputTask.groovy

```diff
@@ -119,7 +119,7 @@ class BugsnagVariantOutputTask extends DefaultTask {
             }
 
             // Get the Bugsnag API key
-            apiKey = getApiKey(metaDataTags, ns)
+            apiKey = getManifestMetaData(metaDataTags, ns, BugsnagPlugin.API_KEY_TAG)
             if (apiKey == null) {
                 project.logger.warn("Could not find apiKey in '$BugsnagPlugin.API_KEY_TAG' " +
                     "<meta-data> tag in your AndroidManifest.xml or in your gradle config")
```

Comment:
This line still mentions looking for the API key in gradle config

Reply:
Updated warning message to only refer to the AndroidManifest.